### ISSUE TITLE
crash on minecraft 1.16.1 + optifine + fabric

### ISSUE THREAD

**Issue by kidzkece** [Wurst-Imperium/Wurst7#190]
## Description
What's wrong?

## Steps To Reproduce
Please make a list of steps to reproduce the problem (ideally from a fresh Wurst installation):
1. 
2. 
3. 

## Screenshots / Videos (Optional)
If applicable, add screenshots, videos or other files to help explain the problem.

## Crash Report
Please find your crash report file at "%appdata%/.minecraft/crash-reports", open the file in a text editor and copy-paste its content below.

```

---- Minecraft Crash Report ----
// Hi. I'm Minecraft, and I'm a crashaholic.

Time: 2020-10-12 21:36:24 ICT
Description: Unexpected error

java.lang.NoSuchMethodError: net.minecraft.class_758.setupFog(Lnet/minecraft/class_4184;Lnet/minecraft/class_758$class_4596;FZF)V
    at Not Enough Crashes deobfuscated stack trace.(1.16.1+build.21)
    at net.minecraft.client.render.WorldRenderer.render(WorldRenderer:1501)
    at net.minecraft.client.render.GameRenderer.renderWorld(GameRenderer:1014)
    at net.minecraft.client.render.GameRenderer.render(GameRenderer:691)
    at net.minecraft.client.MinecraftClient.render(MinecraftClient:1021)
    at net.minecraft.client.MinecraftClient.run(MinecraftClient:654)
    at net.minecraft.client.MinecraftClient.handler$zfl000$afterCrashHandled(MinecraftClient:4835)
    at net.minecraft.client.MinecraftClient.run(MinecraftClient:677)
    at net.minecraft.client.main.Main.main(Main:215)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
    at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
    at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Affected level --
  All players: 1 total; [class_746['ALIDIN'/853, l='ClientLevel', x=-282.74, y=64.00, z=-87.16]]
  Chunk stats: Client Chunk Cache: 841, 454
  Level dimension: minecraft:overworld
  Level spawn location: World: (-240,65,-128), Chunk: (at 0,4,0 in -15,-8; contains blocks -240,0,-128 to -225,255,-113), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
  Level time: 3728 game time, 3728 day time
  Server brand: fabric
  Server type: Integrated singleplayer server

-- System Details --
  Minecraft Version: 1.16.1
  Minecraft Version ID: 1.16.1
  Operating System: Windows 10 (amd64) version 10.0
  Java Version: 1.8.0_261, Oracle Corporation
  Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
  Memory: 5628593808 bytes (5367 MB) / 6408896512 bytes (6112 MB) up to 7079985152 bytes (6752 MB)
  CPUs: 4
  JVM Flags: 10 total; -Xmn128M -Xmx6726M -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
  Suspected Mods: Fabric Loader (fabricloader)
  Fabric Mods: accurateblockplacement: Accurate Block Placement 1.0.6
               allstackable: All Stackable 1.2.4.1
               anvil_fix: AnvilFix 1.1.3+build.7-1.16
               appleskin: AppleSkin 1.0.11
               autoconfig1u: Auto Config v1 Updated 3.2.0-unstable
               bettergraves: Better Graves 0.3.0
               blockmeter: Block Meter 1.16.1-fabric0.14.1-1.5.1
               blue_endless_jankson: jankson +
               can-i-mine-this-block: cAn i MiNe thIS bLOCk? 1.0.2
               cardinal-components: Cardinal Components API 2.4.0-nightly.1.16-rc1.build.2
               cardinal-components-base: Cardinal Components API (base) 2.4.1
               cardinal-components-block: Cardinal Components API (blocks) 2.4.0-nightly.1.16-rc1.build.2
               cardinal-components-chunk: Cardinal Components API (chunks) 2.4.1
               cardinal-components-entity: Cardinal Components API (entities) 2.4.0-nightly.1.16-rc1.build.2
               cardinal-components-level: Cardinal Components API (world saves) 2.4.0-nightly.1.16-rc1.build.2
               cardinal-components-util: Cardinal Components API (utilities) 2.4.0-nightly.1.16-rc1.build.2
               cardinal-components-world: Cardinal Components API (worlds) 2.4.0-nightly.1.16-rc1.build.2
               chains: Chains Link 1.1.1
               chunkactivator: Chunk Activator 2.1.0+mc1.16.1
               cloth-basic-math: Cloth Basic Math 0.5.1
               cloth-client-events-v0: Cloth Client Events v0 1.0.4
               cloth-config2: Cloth Config v4 4.6.0
               cotton: Cotton 1.0.5
               cotton-client-commands: Cotton Client Commands 1.0.1+1.16-rc1
               cotton-commons: Cotton Commons 1.0.4
               cotton-config: Cotton Config 1.0.0-rc.7
               cotton-datapack: Cotton Datapack 1.0.5
               cotton-logging: Cotton Logging 1.0.0-rc.4
               cotton-player-events: Cotton Player Events 1.0.2
               craftable_god_apple: Craftable God Apple 1.0.0
               craftingpad: Crafting Pad 1.16-fabric0.13.1-1.0.5
               diggusmaximus: Diggus Maximus 1.3.0-1.16
               dynamicsoundfilters: Dynamic Sound Filters 1.1.2+1.16.1
               easierenchanting: Easier Enchanting 1.2.0
               easiervillagertrading: EasierVillagerTrading 1.16-fabric0.13.1-1.5.2
               enchantedtooltips: Enchanted ToolTips 1.3.1-1.16
               fabric: Fabric API 0.18.0+build.387-1.16.1
               fabric-api-base: Fabric API Base 0.1.3+12a8474c7c
               fabric-biomes-v1: Fabric Biomes (v1) 0.2.7+059ea86602
               fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.4+c6a8ea8902
               fabric-command-api-v1: Fabric Command API (v1) 1.0.8+5ce5339802
               fabric-commands-v0: Fabric Commands (v0) 0.2.0+52d3083602
               fabric-containers-v0: Fabric Containers (v0) 0.1.8+045df74f02
               fabric-content-registries-v0: Fabric Content Registries (v0) 0.1.9+059ea86602
               fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.2+b7f9825d02
               fabric-diagonal-panes: Fabric Diagonal Panes 0.3.0
               fabric-dimensions-v1: fabric-dimensions-v1 1.0.0+a71b305302
               fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.0+e540c58402
               fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.0+16acbe5b02
               fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.0+fe81e12502
               fabric-item-api-v1: Fabric Item API (v1) 1.0.0+16acbe5b02
               fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.0+438f963602
               fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.1+f404f3be02
               fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.0+3fa9f7c502
               fabric-language-kotlin: Fabric Language Kotlin 1.3.72+build.1
               fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.0+e83e061c02
               fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.0+386eb69e02
               fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.2+b764ce9902
               fabric-models-v0: Fabric Models (v0) 0.1.0+dfdb52d602
               fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.5+b50ffc7b02
               fabric-networking-v0: Fabric Networking (v0) 0.1.10+e00ecb5f02
               fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.5.7+2242e77202
               fabric-object-builders-v0: Fabric Object Builders (v0) 0.6.1+a2d21ddd02
               fabric-particles-v1: fabric-particles-v1 0.2.1+0a6f2a7002
               fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.3.8+7dba2d6c02
               fabric-renderer-api-v1: Fabric Renderer API (v1) 0.2.13+eae12eb802
               fabric-renderer-indigo: Fabric Renderer - Indigo 0.3.4+34d6c87102
               fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.1.0+be551d3c02
               fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.3+b7f9825d02
               fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.8+b7084faa02
               fabric-rendering-v0: Fabric Rendering (v0) 1.1.0+5341049002
               fabric-rendering-v1: Fabric Rendering (v1) 1.1.2+346247d702
               fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.2.6+f41e209802
               fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.0+97324d1102
               fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 0.2.7+a4c57d8e02
               fabric-textures-v0: Fabric Textures (v0) 1.0.4+eae12eb802
               fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.1+91fca5cd02
               fabric-tree-chopper: Fabric Tree Chopper 0.5.0
               fabricloader: Fabric Loader 0.9.3+build.207
               fabriczero: Fabric Zero 0.1.2
               fabritone: Fabritone 1.6.1
               gbfabrictools: GBfabrictools 1.2.2+1.16
               grid: Grid 1.16-fabric0.13.1-1.6.1
               harvest: Harvest 1.2.12
               identity: Identity 1.5.1-beta-1.16.1
               infinitevillagertrading: InfiniteVillagerTrading 1.16-fabric0.13.1-1.0.2
               inventoryprofiles: Inventory Profiles 0.4.2
               inventorytabs: Inventory Tabs 0.2.1-1.16.1
               jankson: Jankson 3.0.0+j1.2.0
               kyrptconfig: Kytpt Config 1.1.0-1.16
               lithium: Lithium 0.5.1
               loadcatcher: Entrypoint Catcher 2.0.0
               megane: megane 1.1.0+1.16
               megane-api: megane-api 1.1.0+1.16
               megane-impl: megane-impl 1.1.0+1.16
               megane-runtime: megane-runtime 1.1.0+1.16
               minecraft: Minecraft 1.16.1
               mixedslab: Mixed Slab 1.16.1-fabric-1.1.0.1
               mm: Manningham Mills 2.1
               modmenu: Mod Menu 1.14.5+build.30
               more_features_id: MoreFeature 1.0.8-1.16.1
               notenoughcrashes: Not Enough Crashes 2.1.4+1.16.1
               optifabric: OptiFabric 1.6.4
               org_jetbrains_annotations: annotations 17.0.0
               org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.3.72
               org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.3.72
               org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.3.72
               org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.3.72
               org_jetbrains_kotlinx_kotlinx-coroutines-core: kotlinx-coroutines-core 1.3.7
               org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.3.7
               overpowered_mending: Overpowered Mending 2.2.2
               phosphor: Phosphor 0.6.0+build.7
               playerabilitylib: Pal 1.2.0
               polymorph: Polymorph 0.0.4-1.16.1
               reicheatkeybind: REI Cheat Keybind 1.0.0
               roughlyenoughitems: Roughly Enough Items 4.12.1
               rpg-hud: RPG-Hud 3.8.1
               shuffle: Shuffle 1.3.0
               silky_spawners: Silky Spawners 2.0.0+build.1-1.16
               simple_backpack: SimpleBackpack 1.0.26
               simplexterrain: Simplex Terrain 0.6.5
               spinnery: Spinnery 3.1.14
               staffofbuilding: Staff of Building 1.2.0-1.16
               stepup: StepUp 1.16-fabric0.13.1-1.0.3
               trinkets: Trinkets 2.6.3
               waila: Hwyla 1.9.22
               worldedit: WorldEdit 7.2.0-beta-03+94ce464
               wurst: Wurst Client v7.6-MC1.16.1
               xaerominimap: Xaero's Minimap 20.20.1
               xaeroworldmap: Xaero's World Map 1.10.1
  Launched Version: fabric-loader-0.9.3+build.207-1.16.1
  Backend library: LWJGL version 3.2.2 build 10
  Backend API: Radeon RX 570 Series GL version 4.6.14736 Compatibility Profile Context 20.9.1 27.20.12029.1000, ATI Technologies Inc.
  GL Caps: Using framebuffer using OpenGL 3.0
  Using VBOs: Yes
  Is Modded: Definitely; Client brand changed to 'fabric'
  Type: Client (map_client.txt)
  Resource Packs: vanilla (incompatible), virtual/fabric-diagonal-panes:models
  Current Language: English (US)
  CPU: 4x AMD Ryzen 3 2200G with Radeon Vega Graphics
  Client Crashes Since Restart: 2
  Integrated Server Crashes Since Restart: 0

```

**Please note:** If the game did not crash, please hold down F3+C for 10 seconds to generate a crash report. Even when a bug doesn't cause the game to crash, this file still contains useful information that can help us to find and fix the problem. Because of this, please always include a crash report.

**Comment by potates66** [comment]
try a different map mod. I think that might be it. I had to use a different map mod for fabric

**Comment by Alexander01998** [comment]
This has been fixed in https://github.com/Wurst-Imperium/Wurst7/commit/e5183965d9a37be1de89060c059f513df7a94888 and will be released in either v7.6.2 or v7.7.
